model/model.go

```diff
@@ -0,0 +1,15 @@
+package model
+
+import "time"
+
+type DepartingTrainId struct {
+	ID string `json:"id"`
+}
+
+type Cancellation struct {
+	ID                 string
```

Comment:
each cancellation has a unique key?

Reply:
Yeah , though its slightly useless right now, but the trainID is recycled every day ( this comes from national rail) so I couldn't use that, I have debating if a relational DB is actually right for this to be honest.